components/mediation/extensions/org.wso2.micro.integrator.transport.handlers/pom.xml

```diff
@@ -53,7 +53,7 @@
                             javax.servlet.http;version="${imp.pkg.version.javax.servlet}",
                             org.apache.commons.logging; version="${import.package.version.commons.logging}",
                             org.apache.axis2.*; version="${imp.pkg.version.axis2}",
-                            org.yaml.snakeyaml.*; version="${orbit.snakeyaml.import.version.range}",
+                            org.yaml.snakeyaml.*; version="${orbit.snakeyaml.import.version}",
```

Comment:
Can't we put a version range instead here?

Reply:
done